webapp/components/admin_console/legal_and_support_settings.jsx

```diff
@@ -113,7 +113,7 @@ export default class LegalAndSupportSettings extends AdminSettings {
                     helpText={
                         <FormattedMessage
                             id='admin.support.helpDesc'
-                            defaultMessage='Link to help documentation from team site main menu. Typically not changed unless your organization chooses to create custom documentation.'
+                            defaultMessage='Link to help documentation from the Main Menu. Typically not changed unless your organization chooses to create custom documentation. Leave this field blank to hide the option from the Main Menu.'
```

Comment:
The URL for the Help link on the Mattermost login page, sign-up pages, and Main Menu. If this field is empty, the Help link is hidden from users.